src/api/api.js

```diff
@@ -11,6 +12,8 @@ import {
 } from '../constants';
 import { getStore } from '../store';
 import { dataURLtoFile } from '../helpers';
+import { LOGOUT } from '../components/Account/Login/Login.constants';
```

Comment:
it's better to import the action directly? 
`import { logout } from '../components/Account/Login/Login.actions.js'`

Reply:
you are right, fixed